data/categories.txt

```diff
@@ -3207,7 +3207,7 @@ fi:Puhelin
 sw:Simu
 
 amenity-toilets
-en:3Toilet|wc|U+1F6BD|U+1F6BE|U+1F4A9|U+1F6BB|U+1F6B9|U+1F6BA
+en:3Toilet|wc|4restroom|4bathroom|U+1F6BD|U+1F6BE|U+1F4A9|U+1F6BB|U+1F6B9|U+1F6BA
```

Comment:
Согласен, Макс, ты молодец! Поправлю.

> On 16.8.2016, at 02.14, mpimenov <notifications@github.com> wrote:
> 
> In data/categories.txt:
> 
> > @@ -3207,7 +3207,7 @@ fi:Puhelin
> >  sw:Simu
> >  
> >  amenity-toilets
> > -en:3Toilet|wc|U+1F6BD|U+1F6BE|U+1F4A9|U+1F6BB|U+1F6B9|U+1F6BA
> > +en:3Toilet|wc|4restroom|4bathroom|U+1F6BD|U+1F6BE|U+1F4A9|U+1F6BB|U+1F6B9|U+1F6BA
> 4restroom точно норм? Может, 5?
> Не забывай, что есть 3restaurant
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>